src/skeletal_model.c

```diff
@@ -822,14 +824,6 @@ int  Anim_SetNextFrame(struct ss_animation_s *ss_anim, float time)
     ss_anim->frame_time = (ss_anim->frame_time >= 0.0f) ? (ss_anim->frame_time) : (0.0f);
     ss_anim->frame_time += time;
     new_frame = ss_anim->frame_time / ss_anim->period;
-    if((ss_anim->current_animation == ss_anim->prev_animation) && 
```

Comment:
I do not know what it was

Reply:
i used the skeletal_model of this repo, so not by me :x

Reply:
I know, it is my code, but I forgot why I had add that =)